src/DynamoCore/Models/DynamoModelDelegates.cs

```diff
@@ -11,6 +11,14 @@ namespace Dynamo.Models
     public delegate void NodeHandler(NodeModel node);
     public delegate void WorkspaceHandler(WorkspaceModel model);   
     public delegate void ActionHandler(Action action);
+
+    /// <summary>
+    /// Delegate, that is mostly used in workspace events.
+    /// When user tries to do something with node.
```

Comment:
remove : // When user tries to do something with node./// E.g. select it, drag it etc.